docs/development.rst

```diff
@@ -158,66 +141,66 @@ handled.
 
 Adding a Theory to the codebase is done by following these steps:
 
-1. Tests: Add a test file ```pysmt/test/test_<theory>.py```, to
-demonstrate the use for the theory (e.g., ```pysmt/test/test_bv.py```).
+1. Tests: Add a test file ``pysmt/test/test_<theory>.py``, to
+    demonstrate the use for the theory (e.g., ``pysmt/test/test_bv.py``).
```

Comment:
Careful here, the indentation has a semantic, and actually means that it is a new line. If you look at the rendered version with Sphinx, you'll see that there is a new-line after to.